miraclelinux/meta-debian-extended
# Add edac-utils package

# Purpose of pull request

edac_utils

This pr adds edac_utils package based on meta-openembedded's /meta-networking/recipes-connectivity/firewalld/firewalld_0.6.3.bb

# Test
## How to test

Check the existence of the component.
```
root@qemuarm64:~# find / -name "libedac*" -type f
root@qemuarm64:~# which edac-util
root@qemuarm64:~# which edac-ctl
```
Execute the edac-util and edac_core commands.

## Test result

Find Components
```
root@qemuarm64:~# find / -name "libedac*" -type f
/usr/lib/libedac.so.1.1.1
root@qemuarm64:~# which edac-util
/usr/bin/edac-util
root@qemuarm64:~# which edac-ctl
/usr/sbin/edac-ctl
```
### edac-util
```
root@qemuarm64:~# edac-util
edac-util: Error: No memory controller data found.
root@qemuarm64:~# edac-util -v
edac-util: Error: No memory controller data found.
root@qemuarm64:~# edac-util -h
Usage: edac-util [OPTIONS]
  -h, --help           Display this help
  -q, --quiet          Display only non-zero error counts and fatal errors
  -v, --verbose        Increase verbosity. Multiple -v's may be used
  -s, --status         Display EDAC status
  -r, --report=REPORT  Display EDAC error report REPORT

Valid REPORT types are default, simple, full, ue, ce

root@qemuarm64:~# edac-ctl --status
[  204.898364] random: edac-ctl: uninitialized urandom read (4 bytes read)
edac-ctl: drivers not loaded.
```
### edac-ctl
```
root@qemuarm64:~# edac-ctl
[   46.142061] random: edac-ctl: uninitialized urandom read (4 bytes read)
Usage: edac-ctl [OPTIONS...]
 --quiet            Quiet operation.
 --mainboard        Print mainboard vendor and model for this hardware.
 --status           Print status of EDAC drivers.
 --print-labels     Print Motherboard DIMM labels to stdout.
 --register-labels  Load Motherboard DIMM labels into EDAC driver.
 --delay=N          Delay N seconds before writing DIMM labels.
 --labeldb=DB       Load label database from file DB.
 --help             This help message.
root@qemuarm64:~# edac-ctl --status
[  260.924662] random: edac-ctl: uninitialized urandom read (4 bytes read)
edac-ctl: drivers not loaded.
```
